protos/orchestrator_service.proto

```diff
@@ -57,14 +57,21 @@ message ParentInstanceInfo {
     OrchestrationInstance orchestrationInstance = 4;
 }
 
+message TraceContext {
+    string traceID = 1;
+    string spanID = 2;
+    google.protobuf.StringValue traceState = 3;
+}
+
 message ExecutionStartedEvent {
     string name = 1;
     google.protobuf.StringValue version = 2;
     google.protobuf.StringValue input = 3;
     OrchestrationInstance orchestrationInstance = 4;
     ParentInstanceInfo parentInstance = 5;
     google.protobuf.Timestamp scheduledStartTimestamp = 6;
-    google.protobuf.StringValue correlationData = 7;
+    TraceContext parentTraceContext = 7;
+    google.protobuf.StringValue orchestrationSpanID = 8;
```

Comment:
For ExecutionStartedEvent, the `parentTraceContext` field is the trace context of the client that started the orchestration. The `orchestrationSpanID` is the span ID that gets generated after the orchestration starts running (and re-hydrated for each replay). I thought it made sense to make this a separate property from the parent trace context since it's unique to execution start events.

Reply:
Thanks for the explanation! Would it be worth adding this information as a comment for future reference?